src/helpers/log-metrics.ts

```diff
@@ -17,16 +17,19 @@ const logMetrics = {
     return process.hrtime.bigint();
   },
 
-  end(start: bigint, flow: string, flowType: string, id: string) {
+  end(start: bigint, flow: string, flowType: string, id: string, enableLog: boolean = true) {
     const ns = process.hrtime.bigint() - start;
     const s = Number(ns) / 1e9;
     if (prometheusTimeEnabled()) logMetrics.prometheusEndClient().observe({ flow, flowType }, s);
     if (newRelicEnabled()) getNewRelic().recordMetric(`Custom/${flowType}/${flow}`, s);
-    logger.debug(`[${id}] TIME_LOGGING[${flowType}][${flow}] ${s.toFixed(3)} s`);
+
+    if (enableLog) logger.debug(`[${id}] TIME_LOGGING[${flowType}][${flow}] ${s.toFixed(3)} s`);
+    else logger.trace(`[${id}] TIME_LOGGING[${flowType}][${flow}] ${s.toFixed(3)} s`);
```

Comment:
🔧 we should update the param variable to determine the log level instead.

Reply:
Done. I preferred to be a boolean again as it is easier for those two levels